googlecloud-plugins/widgets/BigQuery-batchsource.json

```diff
@@ -0,0 +1,48 @@
+{
+  "metadata": {
+    "spec-version": "1.0"
+  },
+  "configuration-groups": [
+    {
+      "label": "BigQuery",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Project ID",
+          "name": "projectId"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Json Key File Path",
+          "name": "jsonFilePath"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Google Storage Temp Path",
+          "name": "tempBuketPath"
```

Comment:
I know I changed a variable name here: it needs changing elsewhere to match. Not a good practice to use mis-spelled variable names.
```
"label": "Google Temporary Storage Path",
"name": "tempBucketPath"